.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+#### Context
+
+* *node version*:
+* *glue version*:
+* *hapi version*:
+* *any other relevant information*:
+
+#### What are you trying to achieve or the steps to reproduce ?
+
+Describe your issue here, include manifests and plugin objects if needed.
+
+```js
+const manifest = {};
+```
+
+#### Which result you had ?
```

Comment:
"What result did you get ?"

Reply:
Seems the other hapi repos share that odd phrasing...

Reply:
Yeah, I debated with myself on fixing the phrasing or staying consistent. I don't have strong opinions in either direction.

Reply:
I'd fix it - the main hapi repo has corrected it.

Reply:
Done